pkg/selector/azure/selector.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package azure
+
+import (
+	"context"
+
+	"github.com/chaos-mesh/chaos-mesh/api/v1alpha1"
+)
+
+type SelectImpl struct{}
+
+func (impl *SelectImpl) Select(ctx context.Context, azureSelector *v1alpha1.AzureSelector) ([]*v1alpha1.AzureSelector, error) {
```

Comment:
It seems the selector will not be used.

Reply:
But you have to write it. It seems odd when a chaos do not need selector comes to the chaos-controller now.

Reply:
@Hexilee , whether we need to allow some chaos work without a selector like GCP-Chaos, Azure-Chaos, AWS-Chaos, since they can select by themselves ?

Reply:
If it is not written here, it is not only strange, but there may be some problems? So I kept this piece for now.